src/utils/cookieUtils.ts

```diff
@@ -0,0 +1,43 @@
+const includeDomain = process.env.NODE_ENV !== "test"
+
+export const getTimeLeft = (expTime) => {
+  console.log(expTime)
+  return (new Date(expTime).getTime() - new Date().getTime()) / 1000
+}
+/**
+ * setCookieWithMaxAge(sKey, expiration, value)
+ * set a cookie
+ * @param {string} sKey - The name of the cookie
+ * @param {string} expiration - when to expire the cookie. Defaults to immediately expire
+ * @param {string} value - The value of the cookie, defaults to an empty string
+ */
+
+const setCookieWithExpiration = (sKey, expiration, value = "") => {
+  document.cookie =
+    `${sKey}=${value}; expires=${expiration || Date.now()}; ` +
+    (includeDomain ? "path=/; domain=.nypl.org;" : "")
+}
+
+export const incrementTime = (minutes, seconds = 0) => {
```

Comment:
Small nit but can you rename this to something like getIncrementedTime or getIncrementedCurrentTime or something like that? I think it would make it more legible since to me incrementTime suggests that there might be a side effect to an existing value.